src/modules/rewriter/export.ts

```diff
@@ -11,70 +12,58 @@ import { isVerbose } from '../../utils'
 import {
   accountAndWorkspace,
   deleteMetainfo,
-  MAX_ENTRIES_PER_REQUEST,
   MAX_RETRIES,
   METAINFO_FILE,
-  progressBar,
   saveMetainfo,
   sleep,
   RETRY_INTERVAL_S,
   showGraphQLErrors,
 } from './utils'
 
 const EXPORTS = 'exports'
+
 const [account, workspace] = accountAndWorkspace
 
+const COLORS = ['cyan', 'black', 'red', 'green', 'yellow', 'blue', 'magenta', 'cyan', 'white', 'gray']
 const FIELDS = ['from', 'to', 'type', 'endDate']
 
-const generateListOfRanges = (indexLength: number) =>
-  map(
-    (n: number) => [n * MAX_ENTRIES_PER_REQUEST, Math.min((n + 1) * MAX_ENTRIES_PER_REQUEST - 1, indexLength)],
-    range(0, Math.ceil(indexLength / MAX_ENTRIES_PER_REQUEST))
-  )
-
 const handleExport = async (csvPath: string) => {
-  const rawRoutesIndexFiles = await rewriter.routesIndexFiles()
-  if (!rawRoutesIndexFiles) {
-    log.info('No data to be exported.')
-    return
-  }
-  const routesIndexFiles = prop('routeIndexFiles', rawRoutesIndexFiles)
   const indexHash = await createHash('md5')
```

Comment:
No need for `await` here:
https://nodejs.org/api/crypto.html#crypto_crypto_createhash_algorithm_options

```suggestion
  const indexHash = createHash('md5')
```